webapp/channels/src/components/global_header/right_controls/saved_posts_button/saved_posts_button.tsx

```diff
@@ -35,7 +35,7 @@ const SavedPostsButton = (): JSX.Element | null => {
         <Tooltip id='recentMentions'>
             <FormattedMessage
                 id='channel_header.flagged'
-                defaultMessage='Saved posts'
+                defaultMessage='Saved Messages'
```

Comment:
@matthewbirtch Is this capitalization correct? Looking at the app right now, I notice that some are capitalized like "Saved posts" and "Channel files" while others are "Channel Info" and "Channel Notes"

Reply:
@hmhealey we have an inconsistency across the app when it comes to capitalization. I would suggest that's a separate effort to tackle outside of this PR. Technically, we are supposed to use Sentence case for titles

Reply:
In that case, could you change the capitalization of these back, @AshishDhama? Like this one should be "Saved messages"